src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageItem.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel.Design;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Collections;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.Options
+{
+    public class PackageItem
```

Comment:
I made packageItem to make it easier to format mappings in the UI (options page ListView)

Reply:
Don't view models need to implement `INotifyPropertyChanged`, so that WPF knows to update the UI when/if values change?

Reply:
> Don't view models need to implement INotifyPropertyChanged, so that WPF knows to update the UI when/if values change?

I think the unique thing about this VM is that it's only binding once. The `PackageItem` is created with an immutable id and the reference for this collection is also immutable.  If we supported editing a mapping, and modifying `ID`, then  we'd have to raise property change events.

I think this implementation works without that since it adds to an observable, which itself will raise property- and collection- changed events.
https://github.com/NuGet/NuGet.Client/pull/4711/files#diff-e0d5570d37e7bf0b411d013d34c8e4576c862bb5c32d5c849288651a74957631R102